packages/zk-eddsa-event-ticket-pcd/src/ZKEdDSAEventTicketPCD.ts

```diff
@@ -459,8 +459,7 @@ export function getProveDisplayOptions(): ProveDisplayOptions<ZKEdDSAEventTicket
       fieldsToReveal: {
         argumentType: ArgumentTypeName.ToggleList,
         displayName: "",
-        description:
-          "...with the following information. The rest will remain private to you."
+        description: "The following information will be revealed"
```

Comment:
I suggest we modify the copy for anonsend where no ticket information is revealed. As is, this confuses me whether there is a bug not showing me the information being revealed or it is in fact telling me no information is revealed for anonsend.

I am okay with simplification here for the ticket auth case. However, I also think it is important that, in terms of education/beating our drum, users are fully aware what kind of benefits they are getting "zero knowledge". Can we ask devconnect folks when testing "what do you think happened here?" and whether we need to spell it out more? For example, we could also say "Only the following information will be revealed:"

Reply:
Agreed, I think that's noted in #881